ledger/accountdb.go

```diff
@@ -196,6 +196,11 @@ type persistedOnlineAccountData struct {
 	round basics.Round
 }
 
+type wrappedPersistedOnlineAccountData struct {
```

Comment:
Sorry for asking correction of my own idea of wrapping `persistedOnlineAccountData` but I see now it does not look great, so I suggest to put `updRound` into `persistedOnlineAccountData` and use `updRound` in the caching logic instead of `round` (do not forget to populate it in `onlineAccountsNewRound` and in `loadOld` and maybe somewhere else.
Rationale:
1. For the caching purpose, the preparation logic re-assigns `round` to be `updRound` and this looks confusing.
2. updRound is part of DB data and makes sense saving it into `persistedOnlineAccountData`